### ISSUE TITLE
Pane delimiters in tmux disappear for certain fonts at small sizes

### ISSUE THREAD

**Issue by abrodkin** [microsoft/terminal#9235]
# Environment

```none
[Environment]::OSVersion

Platform ServicePack Version      VersionString
-------- ----------- -------      -------------
 Win32NT             10.0.18363.0 Microsoft Windows NT 10.0.18363.0

Windows Terminal version: 1.6.10412.0

Ubuntu 20.040 in WSL2
```

# Steps to reproduce

1. Install "DejaVu Sans Mono for Powerline" font in the system (for example from https://github.com/powerline/fonts/tree/master/DejaVuSansMono), or non-Powerline original DejaVu (it behaves just the same as its Powerline derivative)
2. Set it as a default font for Windows Terminal or just for WSL and of size 9 (that's the only size when issue manifests):
    ```
    "fontFace" : "DejaVu Sans Mono for Powerline",
    "fontSize" : 9,
    ```
3. Start Ubuntu in WSL tab
4. Start `tmux`
5. Split window into *vertical* panes:
    ```
    Ctrl+B
    :split-window -v
    Enter
    ```

# Expected behavior

Thin line between panes.
![image](https://user-images.githubusercontent.com/1618098/108620279-2f93b080-743c-11eb-83c2-34a8d0c160ac.png)

The delimiter line (which is just a unicode symbol "─" (copy-pasted here from the terminal terminal) is displayed unless specified above combo of font & size are used.

I.e the following setup changes make the delimiter to appear:
1. The same font, but sizes != 8 (screenshot above is with size = 10)
2. Another font (tried "Cascadia Code PL", "Source Code Pro for Powerline", "Ubuntu Mono derivative Powerline", "Roboto Mono for Powerline")
3. Display scaling of 150% (on 100 & 125% the issue still happens), thus on HiDPI displays with enabled by default scaling you won't see it.

# Actual behavior
No *visible* line between panes:
![image](https://user-images.githubusercontent.com/1618098/108620257-02470280-743c-11eb-9055-382079402a49.png)

**Comment by zadjii-msft** [comment]
Huh. Weird. This one must just get shrunk into oblivion with the stretching we're doing. @miniksa as an FYI of a bug I'm throwing on you for 2.0 ☺️

**Comment by miniksa** [comment]
> Huh. Weird. This one must just get shrunk into oblivion with the stretching we're doing. @miniksa as an FYI of a bug I'm throwing on you for 2.0 ☺️

Gee thanks. :P

**Comment by arielsalvo** [comment]
Hi!
I think I had the same problem just now.
Hope this helps...

Disabled history, scrollbar and scroll to input (I don't need any of those when using tmux, at least in my setup)
After that, the issue seems to be resolved.

**Comment by bitroniq** [comment]
I also have this issue, but only with settings as below:

![image](https://user-images.githubusercontent.com/1032724/145033030-bd305377-41ce-4362-8e39-cd3a661e09d1.png)

![image](https://user-images.githubusercontent.com/1032724/145033334-60222280-66c5-43fc-a9d2-9e36e5e29d15.png)

![image](https://user-images.githubusercontent.com/1032724/145033381-c65b0a48-e29b-49da-bd8b-d4af9b5cc610.png)

When I change the size to smaller than 9 or bigger, then it's OK:

![image](https://user-images.githubusercontent.com/1032724/145033458-59b9d6cb-847c-449d-a6ed-8634d5a5db28.png)

![image](https://user-images.githubusercontent.com/1032724/145033509-c43a3d5a-ddad-4ae1-8f88-abba644de92a.png)

**Comment by zadjii-msft** [comment]
Also seems to happen with Lucidia Console, 8pt, 100% DPI (see #13011). Might just have been fixed for the atlas engine in #13549

**Comment by fizz125** [comment]
I've been watching this since it was affecting me as well, and can confirm that this is fixed with the 1.16 preview release

(Incidentally, it uncovered a new text rendering issue, but I'll file that in a new bug)
Font: Lucida Console, 10pt
Before: (v1.14.2281.0)
![lucidaconsole_10pt_1 14](https://user-images.githubusercontent.com/1494294/190280385-c15297b9-840e-43c8-b8b4-ade3320e50bd.png)

After: (v1.16.2523.0)
![lucidaconsole_10pt_1 16](https://user-images.githubusercontent.com/1494294/190280488-5dc0b112-2f01-4196-97bf-da5e6e896879.png)

**Comment by zadjii-msft** [comment]
Oh I can't read, yea looks like the new renderer did fix this
